### ISSUE TITLE
Specify Encoding or stick to ASCII

### ISSUE THREAD

**Issue by llchan** [ipython/ipyparallel#226]
I'm getting encoding-related issues with the 6.0.0 release:
```
SyntaxError: Non-ASCII character '\xc3' in file /path/to/lib/python2.7/site-packages/ipyparallel/util.py on line 482, but no encoding declared; see http://www.python.org/peps/pep-0263.html for details
```

The line is [here](https://github.com/ipython/ipyparallel/blob/c6f0ff99bc6ac08371d5227c9dea9c58fd1faaf4/ipyparallel/util.py#L482), and it's just a comment saying this:
```python
        # No more naïve datetime objects!
```

I think we just need to specify the file encoding or change that to "naive".

**Comment by minrk** [comment]
Indeed. I forget about Python 2 sometimes. I'm a bit surprised the tests are passing with this..., but it should be fixed by #227

**Comment by minrk** [comment]
Just released 6.0.1 with this fix
